src/app/pages/HomePage/GraphTooltip/SapphireGraphTooltip.tsx

```diff
@@ -0,0 +1,79 @@
+import {
+  GraphTooltip,
+  GraphTooltipDescriptionText,
+  GraphTooltipExtendedProps,
+  GraphTooltipHeaderText,
+  GraphTooltipIcon,
+  GraphTooltipStyled,
+  GraphTooltipText,
+} from './index'
+import { FC } from 'react'
+import AccessTimeIcon from '@mui/icons-material/AccessTime'
+import { COLORS } from '../../../../styles/theme/colors'
+import Typography from '@mui/material/Typography'
+import { useTranslation } from 'react-i18next'
+import { useTheme } from '@mui/material/styles'
+import useMediaQuery from '@mui/material/useMediaQuery'
+import { GraphTooltipMobile, GraphTooltipMobileProps } from './GraphTooltipMobile'
+
+const SapphireGraphTooltipContent: FC = () => {
```

Comment:
This is basically the same as CipherGraphTooltip (different labels). I think from maintenance standpoint it's better to have "ActiveParaTimeTooltip", "DisabledParaTimeTooltip" instead of dedicated component for every paraTime.